framework/src/media/audio/audio_manager.c

```diff
@@ -438,12 +438,11 @@ static unsigned int resample_stream_in(audio_card_info_t *card, void *data, unsi
 			return AUDIO_MANAGER_RESAMPLE_FAIL;
 		}
 
+		used_frames += srcData.input_frames_used;
 		if (srcData.output_frames_gen > 0) {
 			resampled_frames += srcData.output_frames_gen;
-			used_frames += srcData.input_frames_used;
-			medvdbg("Record resampled in:%d/%d, out:%d\n", used_frames, frames, resampled_frames);
-		} else {
-			meddbg("Wrong output_frames_gen : %d\n", srcData.output_frames_gen);
+		} else if (frames != used_frames) {
```

Comment:
Please check this condition one more time. I think output_frames_gen must greater than 0.

Reply:
`output_frames_gen == 0` happens only when there's not enough input frames for converting.
we need to handle such case, so output_frames_gen may be 0.
I think it's better than return an error code, because it's a normal case.

Reply:
got it.